### PR TITLE
Adilet/prev and next buttons

### PR DESCRIPTION
**Pull request by adilet-nasirov**
added buttons, and changed stylings of btns

### CODE REVIEW

**Comment by adilet-nasirov** [comment]
@tagaism  can u check and merge, for now added just buttons, didn't go to functions yet

**Review by tagaism — CHANGES_REQUESTED** [review]
First, create this branch from `main`.

File: index.html [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
-        <video id="background-video" autoplay loop muted poster="images/static.jpeg">
-            <source src="images/crowd.mp4" type="video/mp4">
-        </video>
+      <video
+        id="background-video"
+        autoplay
+        loop
+        muted
+        poster="images/static.jpeg"
+      >
```
Comment (tagaism) [inline index.html]: @adilet-nasirov please do not do unnecessary changes.
